wc_langPacks/wc_langPack_EN.js

```diff
@@ -45,9 +45,9 @@ wc_addLanguagePack({
     // 5+3, 5+4 => nearly
     if (
       (settings.fuzzyTime == 'both' || settings.fuzzyTime == 'after') &&
-      m % 5 <= 2
+      m % 5 <= 2 && m % 5 != 0
     ) {
-      ret += 'JUST WAS ';
+      ret += 'IS JUST AFTER ';
     } else {
       ret += 'IS ';
     }
```

Comment:
Can you move the IS out to line 26?
Then you can remove the else.
Thanks for catching the 0 minutes bug.

Reply:
Yes, great idea. Much more efficient :) I've done this to my local copy and it works fine. Do I need to create a new PR for this?

Reply:
You just push to your PR branch - this automatically updates the PR.

Reply:
I *think* I have done this :)